jaxrs/src/main/java/com/cerner/beadledom/jaxrs/JaxRsParamConditions.java

```diff
@@ -43,10 +43,9 @@ private JaxRsParamConditions() {
   public static void checkParam(boolean expression, Object errorMessage) {
```

Comment:
I am curious why does this have to be an `Object`. Why not String ? It seems like we are converting explicitly to string.